test/unit_test_impl.hpp

```diff
@@ -333,7 +333,14 @@ bool engine::run(bool colorize,
     // psv == pipe-separated-values
     strvec result;
     if (psv != ".*") {
-      split(result, psv, "|", token_compress_on);
+      // Quick & dirty fix
```

Comment:
The comment makes only sense in the diff, so I'd either comment the previous line or remove the comment entirely.